### DISCUSSION TITLE
Background noise and output parameters

### DISCUSSION THREAD

**Discussion by lidiosjr** [Universite-Gustave-Eiffel/NoiseModelling#646] [General]
Hi All,
We just started learning NoiseModelling, and we want to trial this powerful tool for assessing noise at mining sites.
However, we got two questions that we couldn't answer by using the tutorials and chat discussions.
Here they are:
- Is the software able to handle background (baseline) noise measures within the model?
- Regarding statistical parameters, can the model generate results in L10, Lmax, or is it in Leq?
Regards,
Lidio

**Comment by nicolas-f** [comment]
Hi

>     * Is the software able to handle background (baseline) noise measures within the model?

The noise level in receivers are only based on the sound source you provide. You can add a background noise by doing an energetic sum of the background noise into the receiver level.
 
>     * Regarding statistical parameters, can the model generate results in L10, Lmax, or is it in Leq?

With noise_level_from_source and noise_level_from_traffic only Leq value is computed (Ld Le Ln).

 There is a tutorial with time information here so you can compute L10 and Lmax:

https://noisemodelling.readthedocs.io/en/latest/Matsim_Tutorial.html

Are you using NoiseModelling for road traffic noise ?

**Comment by lidiosjr** [comment]
Hi Nic,
Thanks for your reply.

We are trying to use NoiseModelling for noise assessments of mining sites. Therefore, I'm not sure the tutorial you referred can be applied. Can you confirm?

The deal is that the Legislation here in Western Australia defines the assigned levels in terms of L10, L1 and Lmax. Besides that, these assigned levels are defined based on the receivers and are not associated with the noise source (whether it is a mining site, an industry, or any other noise source). Therefore, those assigned levels should be achieved wherever the sensitive premises/receivers (residencies, hospitals, schools, aged care, among others) are located.

Regards,
Lidio

**Reply by lidiosjr** [reply]
Hi Nic,
If you think we could make a video call with your team to discuss this, let me know.
This would be handy to understand possibilities to explore NoiseModelling.

Regards,
Lidio

**Comment by pierromond** [comment]
Hi @lidiosjr,

Probably this tutorial is the one you need for mining site : https://noisemodelling.readthedocs.io/en/latest/Noise_Map_From_Point_Source.html

About background noise, it so depends to your case study. If it is road traffic background noise, you can use NoiseModelling also, but you will probably do some measurements one site, and theses measurements can also be your background noise in your analysis.

About L10, L1, Lmax, it only depends on the definition of your Lw for each source. If they are Lw1 so you will get Leq1 on each receivers... You can also compute dynamic maps as propose @nicolas-f but it is more difficult.

Feel free to ask if you need more info !

Regards,
